WordPress/WordPressTest/NSStringHelpersTest.m

```diff
@@ -116,4 +116,23 @@ - (void)testIsWordPressComPathWithInvalidProtocols
     }
 }
 
+- (void)testWordCount
+{
+    NSString *testEmptyPhrase = @"";
+    XCTAssert([testEmptyPhrase wordCount] == 0, @"Word count should be zero on a empty string");
+    
+    NSString *testPhraseEnglish = @"The lazy fox jumped over the fence.";
```

Comment:
I normally do one test function for each method unless it's something with
state attached... Any reason to split this?
On Monday, 31 August 2015, Aaron Douglas <notifications@github.com> wrote:

> In WordPress/WordPressTest/NSStringHelpersTest.m
> <https://github.com/wordpress-mobile/WordPress-iOS/pull/4215#discussion_r38303975>
> :
>
> > @@ -116,4 +116,23 @@ - (void)testIsWordPressComPathWithInvalidProtocols
> >      }
> >  }
> >
> > +- (void)testWordCount
> > +{
> > +    NSString *testEmptyPhrase = @"";
> > +    XCTAssert([testEmptyPhrase wordCount] == 0, @"Word count should be zero on a empty string");
> > +
> > +    NSString *testPhraseEnglish = @"The lazy fox jumped over the fence.";
>
> Awesome! I'm being super picky but could you split each of these groupings
> into separate tests? testWordCountEmpty, testWordCountEnglish, etc. I'm a
> bit zealous sometimes about unit tests [image: :grimacing:].
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wordpress-mobile/WordPress-iOS/pull/4215/files#r38303975>
> .
>